_pages/dev/authentication.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Authentication
+feature:
+- name: Authentication
+  spa_version: 221121.1.0
+  cx_version: 2211-jdk21.0
+---
+
+OAuth 2.0 is the default authorization protocol in SAP Commerce Cloud that allows third-party applications to access user data without exposing login credentials. It enhances security and user experience by enabling seamless integration between different services. Spartacus provides out-of-the-box support for OAuth 2.0 authentication methods that involve redirects to an authorization server, primarily the authorization code grant flow and implicit grant flow (*do these need more context/explanation?). The default authentication configuration is authorization code grant, which is the recommended OAuth flow.
```

Comment:
(*do these need more context/explanation?)
I don't think they do, they are oAuth 2.0 specs and if you are dealing with authentication, then you should be familiar with them.  That said, we can add some explanation of each grant flow type if it's necessary.